content/docs/command-reference/repro.md

```diff
@@ -130,8 +127,8 @@ only execute the final stage.
 - `-p`, `--pipeline` - reproduce the entire pipelines that the `targets` belong
   to. Use `dvc dag <target>` to show the parent pipeline of a target.
 
-- `-P`, `--all-pipelines` - reproduce all pipelines, for all the stages present
-  in `DVC` repository.
+- `-P`, `--all-pipelines` - reproduce all pipelines for all the `dvc.yaml` files
+  in present the DVC repository (there can be one in every directory).
```

Comment:
`(there can be one in every directory).` not sure this is needed - it's clear that there can be only one dvc.yaml per directory

Reply:
I mean sure, you can't repeat file names. So true, I'm removing this note here (see a5db93fe).

But actually, we don't mention that DVC supports multiple `dvc.yaml` files here, in `run`, or in the DVC files & dirs guide. Should we?

Reply:
Extracted to #1641